Branch-SDK/Branch-SDK/BNCServerInterface.h

```diff
@@ -18,6 +19,8 @@ static NSString *REQ_TAG_DEBUG_DISCONNECT = @"t_debug_disconnect";
 
 @interface BNCServerInterface : NSObject
 
+@property (strong, nonatomic) BNCPreferenceHelper *preferenceHelper;
```

Comment:
Yes, I want to move toward everything have an *instance* of the PreferenceHelper rather than grabbing the global one. This makes testing easier (read: DI), but it's too much of a hassle to try to get all into this PR, so I just did it in the one, easy spot.